integration-tests/src/test/java/io/druid/tests/security/ITBasicAuthConfigurationTest.java

```diff
@@ -197,6 +202,33 @@ public void testAuthConfiguration() throws Exception
 
     LOG.info("Checking access for user druid99.");
     checkNodeAccess(newUser99Client);
+
+    LOG.info("Checking Avatica query on broker.");
+    testAvaticaQuery("jdbc:avatica:remote:url=" + config.getBrokerUrl() + "/druid/v2/sql/avatica/");
+
+    LOG.info("Checking Avatica query on router.");
+    testAvaticaQuery("jdbc:avatica:remote:url=" + config.getRouterUrl() + "/druid/v2/sql/avatica/");
+  }
+
+  private void testAvaticaQuery(String url)
+  {
+    LOG.info("URL: " + url);
+    try {
+      Properties connectionProperties = new Properties();
+      connectionProperties.put("user", "admin");
```

Comment:
Could you add a test for when the access is _not_ granted? (Wrong password or whatever)

Reply:
Added two tests where authentication fails